tests/zfs-tests/include/tunables.cfg

```diff
@@ -0,0 +1,69 @@
+# This file must be kept up to date with any tunables used by tests.
+
+UNAME=$(uname)
+
+# VARIABLE			FreeBSD tunable			Linux tunable
+cat <<%%%% |
+ADMIN_SNAPSHOT			xxx				zfs_admin_snapshot
```

Comment:
nit: instead of `xxx` let's change this to something more descriptive like `UNSUPPORTED`.